charts/airbyte/Makefile

```
@@ -0,0 +1,34 @@
+RELEASE_NAME := airbyte
+NAMESPACE := airbyte-test
```

```suggestion
NAMESPACE := airbyte
```

Maybe `airbyte` so this can be used for diagnostics/install for non-test cases more easily?